installer/roles/local_docker/defaults/main.yml

```diff
@@ -18,3 +18,5 @@ memcached_image: "memcached"
 memcached_version: "alpine"
 memcached_hostname: "memcached"
 memcached_port: "11211"
+
+compose_start_containers: false
```

Comment:
The default value for this needs to be `true` so that it is not a breaking change to the installer.

Reply:
Absolutely right, I mentioned it in the comment, but left the default. With the new commit it should be fine. Thanks and sorry for wasting your time.

Reply:
@racasoft 

You are not wasting anyone's time. Thank you very much for taking the time to make a contribution to AWX and improve one of the installers.

Reply:
@rascasoft 

You are not wasting anyone's time. Thank you very much for taking the time to make a contribution to AWX and improve one of the installers.